e2e/cypress/integration/messaging/markdown_spec.js

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @messaging @markdown
```

Comment:
Please remove prod. We'll add it once we found this test to be stable for a few runs.
Also, should at `@not_cloud` since `EnableTesting` is currently not possible in the Cloud. This test will run only for on-prem edition.
```suggestion
// Group: @not_cloud @messaging @markdown
```